### ISSUE TITLE
Filtering by tag stopped working for some tags

### ISSUE THREAD

**Issue by EgorBakanov** [usememos/memos#2027]
### Describe the bug

After updating to 14.1, I noticed that some of my old tags no longer work.

### Steps to reproduce

I couldn't find any patterns when the tags break.

### Search by tag
![image](https://github.com/usememos/memos/assets/33174871/04e1407a-aa38-4c21-a08a-2e969195ab32)
### Search by substring
![image](https://github.com/usememos/memos/assets/33174871/9113df97-7e52-4b74-84aa-caccc3a9612b)

**Comment by tranquanghuy0801** [comment]
@EgorBakanov Which page are you searching for this?

**Comment by RaphaelPour** [comment]
Same here!

1. Login to memos and goto home
2. click on a tag at the right side 

Looking at the access log it seems that the range `offset=0` is missing:
```
{"time":"2023-07-26T11:12:55Z","method":"GET","uri":"/api/v1/memo?rowStatus=NORMAL&offset=20&limit=20","status":200,"error":""}
{"time":"2023-07-26T11:12:55Z","method":"GET","uri":"/api/v1/memo?rowStatus=NORMAL&offset=40&limit=20","status":200,"error":""}
{"time":"2023-07-26T11:12:55Z","method":"GET","uri":"/api/v1/memo/stats?creatorUsername=raphael","status":200,"error":""}
{"time":"2023-07-26T11:12:55Z","method":"GET","uri":"/api/v1/memo?rowStatus=NORMAL&offset=60&limit=20","status":200,"error":""}
{"time":"2023-07-26T11:12:55Z","method":"GET","uri":"/api/v1/memo/stats?creatorUsername=raphael","status":200,"error":""}
{"time":"2023-07-26T11:12:55Z","method":"GET","uri":"/api/v1/memo?rowStatus=NORMAL&offset=67&limit=20","status":200,"error":""}
{"time":"2023-07-26T11:12:55Z","method":"GET","uri":"/api/v1/memo/stats?creatorUsername=raphael","status":200,"error":""}
{"time":"2023-07-26T11:12:55Z","method":"GET","uri":"/assets/empty.png","status":200,"error":""}
```

**Comment by boojack** [comment]
Fixed in `v0.14.2`
